images/ceph/Makefile

```diff
@@ -18,9 +18,9 @@ include ../image.mk
 # Image Build Options
 
 ifeq ($(GOARCH),amd64)
-CEPH_VERSION ?= v18.2.4-20240723
+CEPH_VERSION ?= v19.2.0-20240927
 else
-CEPH_VERSION ?= v18.2.4-20240724
+CEPH_VERSION ?= v19.2.0-20240927
```

Comment:
I think this is leading to canary failures related to the external script. I wonder if 19.2.1 might fix it, but I think we're still tracking down the v19 issues with RGW right?

Reply:
still no luck rgw canary test is still failing.

I think should we raise ceph tracker, IMO this is not rook issue as after upgrading rgw tests are failing. Wdyt?

Reply:
Agreed, let's just follow up on it in #14972 as suggested [here](https://github.com/rook/rook/pull/15392#discussion_r1958768172) to unblock this PR.